independent-publisher-connectors/Binance.us/readme.md

```diff
@@ -0,0 +1,23 @@
+# Binance.us
+The Binance platform allows anyone to get crypto market rates and historical trades.  This connector allows a user to pull this data into their app, flow, or report.
+
+## Publisher: Roy Paar
+
+## Prerequisites
+None.
+
```

Comment:
No Obtaining Credentials section. Please review you Readme.md file using this [template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/readme.md).

Reply:
Corrected